tests/Alyx_test.m

```diff
@@ -246,7 +249,27 @@ function test_postWater(testCase)
     end
     
     function test_getFile(testCase)
-      %TODO Add test for getFile method
+      ai = testCase.alyx;
```

Comment:
do you want to add a test for multiple eids in an array? If you think this is unnecessary lmk

Reply:
Yeah I can do this

Reply:
cool, I'll merge when done